examples/index.less

```diff
@@ -153,35 +153,52 @@ header.links {
   tr:nth-child(2n) {
     background: #F9F9F9;
   }
-  .validationTable td:nth-child(1) > code {
-    text-align: center;
-    text-align: center;
-    display: block;
-  }
-  .validationTable td:nth-child(2), .validationTable td:nth-child(3) {
+
+  // validation table styles
+  .validationTable {
     code {
-      display: block;
-      margin-bottom: 10px;
-      margin-top: 10px;
-      padding: 5px;
+      background-color: transparent;
+    }
+    th {
+      border-bottom: none;
       text-align: center;
     }
+    tr {
+      height: 80px;
+    }
+    td:nth-child(1) {
+      vertical-align: top;
+      padding-top: 19px;
+      code {
+        display: block;
+        vertical-align: top;
+        color: #000;
```

Comment:
`code { color: #000; }` is probably the only thing you need to declare under td:nth-child(1)